Makefile

```
@@ -46,7 +46,7 @@ nightly-release: setup-kong-build-tools
 	$(MAKE) build-kong && \
 	$(MAKE) release-kong
 
-release: setup-release
+release:
```

shouldn't this instead be: 

```suggestion
release: setup-kong-build-tools
```